src/python/pants/notes/1.9.x.rst

```diff
@@ -0,0 +1,340 @@
+1.9.x Stable Releases
+=====================
+
+This document describes releases leading up to the ``1.9.x`` ``stable`` series.
+
+1.9.0rc0 (07/17/2018)
+---------------------
+
+New features
+~~~~~~~~~~~~
+
+* Early support for @console_rule. (#6088)
+  `PR #6088 <https://github.com/pantsbuild/pants/pull/6088>`_
+
+API Changes
+~~~~~~~~~~~
+
+* Eliminate obsolete PANTS_ARGS from ci. (#6141)
+  `PR #6141 <https://github.com/pantsbuild/pants/pull/6141>`_
+
+Bugfixes
+~~~~~~~~
+
+* add the target fingerprint to the version of each local dist so that we don't use the first cached one (#6022)
+  `PR #6022 <https://github.com/pantsbuild/pants/pull/6022>`_
+
+* Preserve output directories if process execution failed (#6152)
+  `PR #6152 <https://github.com/pantsbuild/pants/pull/6152>`_
+
+Refactoring, Improvements, and Tooling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Move each EntryState in the graph under its own Mutex (#6095)
+  `PR #6095 <https://github.com/pantsbuild/pants/pull/6095>`_
+
+Python 3 porting (#6062)
+------------------------
```

Comment:
This should be a `~*` heading instead of `-*`.

Reply:
Yes -- I was thinking that this would make sense as a subheading of the "Refactoring, Improvements, and Tooling" section. But looking through other notes now I don't see any other examples of this, so I will change it to a `~*` heading.

Reply:
Done in 4800643!